README.md

```diff
@@ -1,19 +1,26 @@
 # FFC Payment Batch Processor
 
-FFC service to process Siti Agri batch files and send individual transactions as messages.
+## Description
 
-## Prerequisites
+FFC microservice acting as a Siti Agri adaptor to process batch files and send individual transactions as messages to an Azure Service Bus topic.
+
+# Prerequisites
+
+## Software required
 
 - Access to an instance of an
 [Azure Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging/)(ASB).
```

Comment:
I would remove this abbrev. of ServiceBus: it's not a common abbrev, it's not used elsewhere in the README